templates/http-c/metadata/spin-template.toml

```diff
@@ -1,6 +1,7 @@
 manifest_version = "1"
 id = "http-c"
 description = "HTTP request handler using C and the Zig toolchain"
+tags = ["http", "c"]
```

Comment:
@itowlson happy to change the tags for all these templates if you are keen on a particular set.

Reply:
Nah, protocol and language seems like the perfect starter set.